cryptography/src/test/scala/akka/stream/alpakka/cryptography/scaladsl/CryptographicFlowSpec.scala

```diff
@@ -41,11 +41,27 @@ class CryptographicFlowSpec extends WordSpec with Matchers with ScalaFutures wit
           }
         }
       }
+
+      "Be able to handle bytestring chunking" in {
+        val keyGenerator = KeyGenerator.getInstance("AES")
+        val secretKey = keyGenerator.generateKey()
+
+        val byteString = ByteString("byte by byte")
+
+        val res = Source.single(byteString)
+          .mapConcat(bs => bs.toArray.toList)
+          .map(b => ByteString(b))
+          .via(symmetricEncryption(secretKey))
+          .via(symmetricDecryption(secretKey))
```

Comment:
Add a concat between the encryption and decryption and you'll see what I mean. If we - for example - send the encrypted data via a socket or http, we'll get other ByteStrings on the other side. We'd need some frameing to cut out the same ByteStrings for decryption.

Reply:
Oooh you're right, will have a look before the weekend is over!